yarn-project/end-to-end/src/spartan/mempool_limit.test.ts

```diff
@@ -71,11 +71,11 @@ describe('mempool limiter test', () => {
     // set a large pool size so that deploy txs fit and allow blocks with few txs
     const configs = await getSequencersConfig(config);
     originalMinTxsPerBlock = configs[0]?.minTxsPerBlock;
-    await updateSequencersConfig(config, { maxTxPoolSize: 1e9, minTxsPerBlock: 0 });
+    await updateSequencersConfig(config, { maxPendingTxCount: 1e9, minTxsPerBlock: 0 });
```

Comment:
This test must surely no longer work if we haven't updated the values but those values implement a completely different limit criteria now?

Reply:
Completely missed this one, sorry 🤦‍♂️
This is 1GB of transactions, I have changed this number to 10k transactions, which in the worst case should be 1.56GB